integration-test/src/main/java/com/sequenceiq/it/cloudbreak/testcase/e2e/environment/EnvironmentStopStartTests.java

```diff
@@ -100,8 +114,19 @@ public void testCreateStopStartEnvironment(TestContext testContext) {
                 .given(EnvironmentTestDto.class)
                 .when(environmentTestClient.start())
                 .await(EnvironmentStatus.AVAILABLE)
+                .given("dx1", DistroXTestDto.class)
+                .await(STACK_AVAILABLE, RunningParameter.key("dx1"))
+                .awaitForHealthyInstances()
+                .then(this::verifyCmServicesStartedSuccessfully)
```

Comment:
do you want to validate this on every provider?

Reply:
Yes

Reply:
ok